Vungle/src/main/java/com/applovin/mediation/adapters/VungleInitializer.java

```diff
@@ -0,0 +1,71 @@
+package com.applovin.mediation.adapters;
+
+import android.content.Context;
+import androidx.annotation.NonNull;
+import com.vungle.ads.InitializationListener;
+import com.vungle.ads.VungleAds;
+import com.vungle.ads.VungleError;
+import java.util.ArrayList;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class VungleInitializer implements InitializationListener {
```

Comment:
Does MAX allow this for Android?
I think in iOS, MAX forces us to put everything in one file and will complain if there are more than 1 file. @ManojBudumuru-vungle

Reply:
yes generally max wants everything in one class.